packages/babel-plugin-transform-block-scoping/test/fixtures/tdz/const-readonly/output.js

```diff
@@ -0,0 +1,7 @@
+var x = babelHelpers.temporalUndefined;
+maybeCallLater(function () {
+  babelHelpers.temporalRef(x, "x");
+  2, babelHelpers.readOnlyError("x");
```

Comment:
Unrelated to this PR: here it seems numbers can be safely removed.

Reply:
Yeah we can check if the initializer is without side effects.